content/en/docs/configuring-sdk/endpoints.md

```diff
@@ -5,39 +5,79 @@ date: "2020-11-12"
 description: Customizing service client endpoints.
 ---
 
-The {{% alias sdk-go %}} provides the ability to configure a custom endpoint to be used for a service. In most cases you use a pre-configured 
-endpoint for a service. Configuring custom endpoints, lets you do more, such as working with pre-release versions 
-of a service.
-
-A [EndpointResolver]({{< apiref "aws#EndpointResolver" >}}) can be configured to provide custom endpoint resolution
-logic for service clients. You can use a custom endpoint resolver to override a service's endpoint resolution logic
-for all endpoints, or a just specific regional endpoint. Custom endpoint resolver can trigger the service's endpoint 
-resolution logic to fallback if a custom resolver does not wish to resolve a requested endpoint. 
-[EndpointResolverWithOptionsFunc]({{< apiref "aws#EndpointResolverWithOptionsFunc" >}}) can be used to easily wrap functions to satisfy the 
-`EndpointResolverWithOptions` interface.
-
-A `EndpointResolver` can be easily configured by passing the resolver wrapped with
-[WithEndpointResolverWithOptions]({{< apiref "config#WithEndpointResolverWithOptions" >}}) to
-[LoadDefaultConfig]({{< apiref "config#LoadDefaultConfig" >}}), allowing for the ability to override endpoints when
-loading credentials, as well as configuring the resulting `aws.Config` with your custom endpoint resolver.
-
-The endpoint resolver is given the service and region as a string, allowing for the resolver to dynamically drive its
-behavior. Each service client package has an exported `ServiceID` constant which can be used to determine which service
+{{% pageinfo color="warning" %}}
+Endpoint resolution is an advanced SDK topic. By changing these settings
+you risk breaking your code. The default settings should be applicable to
+most users in production environments.
+{{% /pageinfo %}}
+
+The {{% alias sdk-go %}} provides the ability to configure a custom
+endpoint to be used for a service. In most cases, the default
+configuration will suffice. Configuring custom endpoints allows for
+additional behavior, such as working with pre-release versions of a
+service.
+
+## Customization
+
+There are two "versions" of endpoint resolution config within the SDK.
+behavior of endpoint resolution within the SDK:
+1. v1, released alongside the SDK, configured via:
+  * `EndpointResolver`
+1. v2, released in Q3 of 2023, configured via:
+  * `EndpointResolverV2`
+  * `BaseEndpoint`
+
+## V1: `EndpointResolver`
```

Comment:
**question:** Should we move v2 to the top? Inclined to say yes given it's what we want people to use (it effectively supercedes v1 as we know).